engine/src/components/formcomponent.ts

```diff
@@ -87,6 +92,8 @@ export default class FormComponent extends Component {
         [name]: this.getFormValueFromState(state),
       };
     }
+
+    return null;
```

Comment:
do we need to return null? does ts let you `:void` return type for constructors?